### DISCUSSION TITLE
Meeting notes

### DISCUSSION THREAD

**Discussion by lucasmarchd01** [DDMAL/VIM#11] [General]
VIM 7/24/23
- max & taz: the mockup on the github for faceted search 
- drafting a logo as well
- backend discussion in github
	- what data we are going to pull from wikidata
	- where to host audio/visual
- pulling images from wikimedia for orchestral suit instruments etc
- other nav bar entities 
	- browse by grouping
	- contributors page 
- contributors power
	- should their stuff be approved first -even musicbrainz have editors 
	- what happens if facts are contested 
- having the orchestral suite available when the site launches 
	- being tested by languages known in the lab
- translate widget 
- zhanna finishing docker, postgres next
	- docker is working more or less
	- adding nginx and gunicorn
- private docker repo? 
- models implemented roughly after
- musicbrainz translation feature 
- musicbrainz instrument list
- what we need in the footer

**Comment by fujinaga** [comment]
Great notes! Thank you, Lucas!

We can put audio and video at public places, e.g., Internet Archive and Wikidata.

**Reply by fujinaga** [reply]
Thank you, Marisa!

**Comment by maregold** [comment]
VIM 7/31/23
- taxonomy from wikidata
- inclusivity from the get-go 
	- for the logo
	- also for the language list
- Internet Archive 
	- for storage of audiovisual information
- how to pull specific info from wikidata
	- API documentation
	- SPARQL query
- MediaWiki instead of Django 
	- you can roll back version histories (can Django do that)
	- would have the wiki functionality we need for points, discussions, etc.
- sustainability point of view would be best if we didn't have to have a person that moderated everything
	- hybrid approach - the change automatically goes up, but the change is logged and it can be fixed by a moderator 
	- this information is not verified tagline
	- how worried do we need to be worried about abuse 
- point system 
	- positive for good uploads, negative for bad uploads 
	- prioritize changes from good standing accounts, while newer accounts would have that disclaimer
	- group points vs individual
	- points act as motivation 
	- stack overflow - people with higher reputations have some tagline indicating, plus their posts are pushed higher
- how to get a list of languages in the world 
	- historical languages - drawings, paintings, iconography 
- pick a language, shows you the pictures of instruments that are not named yet in that language
	- game element 
- translate widget just for the instruction elements
- translating tags / the verbal desc
	- the original plan was just a picture and name
	-  for characteristics to have a link to somewhere else (like wikipedia)
- we need a little more information for faceted search, to prepare for a large amount of instruments in the long-run
- zhanna updates on the back-end
- wording needs to be the same everywhere
	- Contributors, Instruments, vs Contributor, Instrument
- website should be usable enough that you don't need text to understand what is happening 
- open a website in a country and it opens in that language is separate from a google translate widget 
- want to get away from european control authority for languages 
	- tell us the word, what language the word is in, and the image and we will list it 
- how the name of an instrument changed over time
- date, person, locations - provenance for data

**Comment by maregold** [comment]
VIM 8/7/23
- marisa - taxonomy (musical instruments on wikidata)
	- wikidata works from the bottom up, how well does it work top down
	- make a nicer one
	- taxonomy tree can be part of the website
- django vs mediawiki pros and cons
	- dylan - mediawiki
	- zhanna - django
- verified by trusted user box
	- decided by number of verified of contributions 
- verified vs unverified points
	- if you start uploading junk then the profile will get unverified / points go down
- trusted users would be ranked - more contributions, the higher you are ranked
	- also rank by class - some people may be experts in only one type of musical instrument 
	- you can request a review (via notifications) to have someone that is an expert in that class to review the information to make sure it is correct
	- how do we keep the social media aspect from devolving into chaos 
	- start out with a manual process and then convert it to an automated process for users
- comment system?
- taz - language list
- max - front-end & model
- game element (/game)
- first usage of that name documented in OED (for provenance)
	- add in wikidata?
	- documentation of how the names were used in respective time periods (which would create a documented timeline of use)
- zhanna back-end updates
- open in the language of your browser

**Comment by maregold** [comment]
VIM 8/14/23
- mediawiki vs django debate (can we have the doc link @dchiller)
	- mediawiki is moving out of technical expertise of lab
- descriptions 
	- historical instruments (no images potentially)
	- what do you do when you know a word is a name of an instrument but there is no picture
		- ask them to put a description in wikidata/wikipedia instead so we don't have to host it
		- if it is a new instrument, ask that user to provide a desc and such for when it's uploaded to wikidata and let them argue on wikidata
- utilize the images as a way to avoid having to know the name in a different language (such as english)
- becoming a nice, easy-to-use front for wikidata information 
- the difference between just adding a name and adding a new instrument 
- in conclusion: 70% django 30% mediawiki, but we're going to let it simmer

**Comment by maregold** [comment]
VIM 8/21/23
- Leaning towards Django still
- Need something to show at the project meeting
- where to get descriptions from 
	-  Hornbostel–Sachs number
- Discussion about what landing page/instrument page may look like
	- nested lists
	- dealing with instantiations
		- multiple random fiddles from the royal college of music
	- language translation - automatic translation in django?
- Instruments
	- baroque oboe vs modern oboe
		- wikidata doesnt have baroque oboe entry, but could that be useful for us since they do look different
	- guitar
- how much curation are we doing with images
- filters for search 
	- subclasses from wikidata but still might need some curation 
		- not all instruments have great information attached

**Comment by annamorphism** [comment]
VIM meeting 28/08/2023

Summary of project:
-note: new [goals and purpose page](https://github.com/DDMAL/VIM/discussions/22)
-definition of wikidata/LinkedData
-illustration: “Timpani” as translated by ChatGPT (useful but not authoritative)

Needs:
-Pictures of all instruments that exist or have existed
-The ability to flag what needs a name/image

-Names **and pictures** to be collected via crowdsourcing; starting list from wikidata
[everything on VIM (inc. new items) will end up on wikidata]

-a way to collect secondary names (timpani vs kettledrums), family names (“clarinet”/“b-flat clarinet”)
-other categories to collect: first mentions, iconography

-Gamification: “show pictures with no name in X language”
-possible endpoint: a pictorial front end for musical instruments in wikiData
—
Review of last week+discussion:

Django deemed likely better than MediaWiki 
Plan: create something quick, set up a few models and urls, see what happens.
Dylan will work on this for the project meeting
Will need a list in the various languages, and ability to turn up unnamed things

Descriptions: best to take from wikipedia (thorough, more languages); use links as much as possible

Landing page: ability to select language, show things w/o names and things ready for other modifications (secondary names, challenge/correct names, or even attaching to categories/hierarchies?)

Translation: django is built-in (so no dependencies) but Google is better; will start with Google 

—
Further discussion:

Will consider ourselves “curators” of the musical instruments of wikidata.

Related issues:
-Descriptions: point to wikipedia in the correct language (via wikidata page)
	&nbsp;&nbsp;&nbsp;&nbsp;-some instruments are poorly taxonomized but we use what is there
-Images: use wikimedia/CreativeCommons image; store on Internet Archive (possibly Alliance space?). 
	&nbsp;&nbsp;&nbsp;&nbsp;-what to do when there are multiple images?  may need to curate
-Instrument families:
	&nbsp;&nbsp;&nbsp;&nbsp;-oboes, baroque oboes, english horns
	&nbsp;&nbsp;&nbsp;&nbsp;-guitars>electric guitars>Stratocaster>one guy’s Stratocaster
How do we know what depth of the tree to go to for any given instrument family? Trial and error? Manual removal of garbage? 
	&nbsp;&nbsp;&nbsp;&nbsp;-possibility of using “instance of” musical instrument (i.e. avoid the “subclass” tree)
		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-however e.g. “tin whistle” is only findable through “subclass” 
	&nbsp;&nbsp;&nbsp;&nbsp;-Ich to look into this taxonomy tree 
-Has anybody done a similar project for other wikidata categories (e.g. botany) that could be used as a model? Ich to ponder this as well

**Comment by annamorphism** [comment]
VIM meeting notes, 07/09/2023
Opening with Ich’s homework:
Looked into Wikidata about musical instruments. Some things great, some a mess; @kyrieb-ekat to do an inventory regarding music instruments, "see what junk is in there." May have to keep the garbage or eliminate via trial-and-error--need to suss out what the wikidata culture of clean-up is. 
Do not seem to have other comparable WikiData projects as a precedent. Could find out who has worked on the Hornbostel-Sachs/MIMO info.

Various example labels and queries and their issues (see : https://github.com/DDMAL/VIM/discussions/24):
e.g. "part of", "instance of", "subclass of"—tend to get used interchangeably in practice.
H-S is clearly entered in the Wikidata but plucked box lutes and trumpets right under H-S in the taxonomy 
Over 2000 items with H-S "property", also a few with "part of".

Some IDs with no label need to be found using the H-S no.

Suggestion to filter out specific instances of one instrument using other properties, e.g. "is in collection" "donated by"; or "manufacturer" to filter out e.g. Stratocaster guitars.

Discussion:
-Will we recategorize things received from Wikidata? A: need to understand the culture first. Goal is to create as complete as possible a list of existing musical instrument Wikidata IDs.
-Use of H-S taxonomy: probably useful for a browsable hierarchy. May look into what MIMO uses (is it the same?). Bakes in some sort of structure.
-Q: will we be able to reverse-image-search like google? A: probably not a feature we want/need

Dylan: initial models 
-User model w/ email as log i,; name for display, country, preferred language. Optional about, institution, etc.
 Q: should country be required? may be good for general name provenance or social aspect of site.  Decision to make this optional.
-Instrument model w/wikidata ID, Instrument name+language (or do we just get this off Wikidata directly?), record of what is changed/updated (or is this not stored except as reference), provenance information.
-entry form to add new name or instruments
Discussion regarding language—if we want to query by language don’t we need to store this information? & will we need a model for languages?
-possibility of two models:  instrument w/ single image +ID, second model with “name in language” +foreign key to instument. Every instrument has a list of provided names.
(Extended discussion: https://github.com/DDMAL/VIM/discussions/7#discussioncomment-6939004)

Some mappings needed: languages, instruments, combinations thereof
@dchiller to work with @yinanazhou on designing the pages.

**Comment by kyrieb-ekat** [comment]
From today's (15 Sept) meeting, below is the one drive link to the instruments with images sheet. Please let me know if you have any issues. 

https://mcgill-my.sharepoint.com/:x:/g/personal/kyrie_bouressa_mail_mcgill_ca/EawNG2l2eKdLtN4O__pR4hIBGrvXgdQ6E4Z-YegbGb628g?e=VpzxaB

**Reply by fujinaga** [reply]
This great! Thank you very much!

**Comment by annamorphism** [comment]
Meeting notes 15/09/2023

-Discussion of instrument spreadsheet (see https://github.com/DDMAL/VIM/discussions/11#discussioncomment-7015300)
-Problems encountered: some labels return QID, empty with google graph, only show up in some language search
-some excessively individual IDs (every bell in France)
-some underdescribed ("Ethiopian musical instrument")
-some categories ("Arabic musical instrument") not a subclass of musical instruments
-some duplicates (labelled on sheet)

For next week: have an automatic way of generating a legit instrument name (can be subset of all instruments)
To be presented at project meeting (can discuss the mess).

Discussion of models (see #7)
-storage of taxonomy: what and why. Want aspects for facet search.
-not clear how well-structured/correct the taxonomy is in Wikidata
-do we need "family"?
-note: MIMO taxonomy is expansion of HS to include electric input/output, essentially similar otherwise
-will we put in missing H-S if not specified? 

-Default image: in the AVresourceset, or in instrumentModel?
-latter method has unique default image, avoids impossible states.

-“Name shorthand” is a copy of stuff in the name model—may not be necessary
-name_set: for free in Django, a common join query
-Name model: submissions live here as name.name objects
-Can we capture different sources for instrument name? point to these in NameInLanguage
-Do we want multiple instances of the name in different sources to be their own object or the same object? TBD

-Language: doesn’t provide anything additional but gives stuff for free
-Note Wikidata has a language police about which languages exist

-/languages path: what lives here? a list of things needing labelling in some language?
[nota: should be consistent about singulars/plurals in path names/models]
-will users want to see the full set of stuff that needs labelling
-will also want categorized or faceted list/picture
-Several people to work on front end, have something by project meeting

-Further discussion on front-end specifics here: https://github.com/DDMAL/VIM/discussions/26
-Use-cases need to be refined: see https://github.com/orgs/DDMAL/projects/28/views/1

**Comment by dchiller** [comment]
### Agenda for 21 Sept

#### Spreadsheet of musical instruments
_Last week_
- Problems with using some properties to generate list
> -some labels return QID, empty with google graph, only show up in some language search
> -some excessively individual IDs (every bell in France)
> -some underdescribed ("Ethiopian musical instrument")
> -some categories ("Arabic musical instrument") not a subclass of musical instruments
> -some duplicates (labelled on sheet)
- A spreadsheet of 200-odd initial instruments to include was generated

_To Discuss_
- Do we have an automatic way of generating this initial list of instruments?
- Presentation on generating this list (and problems inherent therein) for the LinkedMusic project meeting?

#### Data structure and backend implementation
_Last week_
- Discussion of how to include instrument taxonomy in database (necessary to provide faceted search); decision to wait on this and stick with existing classification schemes (Hornbostel-Sachs and MIMO) for now
- Discussion of how to capture multiple sources for instrument names
> -Can we capture different sources for instrument name? point to these in NameInLanguage
> -Do we want multiple instances of the name in different sources to be their own object or the same object? TBD

_To Discuss_
- What is the status of implementation of the agreed upon models?

#### Plan for October Meeting
 
_To Discuss_
- Discuss user stories to be refined: https://github.com/orgs/DDMAL/projects/28/views/1

#### Frontend

_To Discuss_
- How can we get started on front-end development to support completion of agreed-upon user stories by October meeting

**Comment by annamorphism** [comment]
VIM meeting, 21.09.2023
See above for meeting agenda.

**Spreadsheet of musical instruments discussion:**
    _Do we have an automatic way of generating this initial list of instruments?_
Not so worried about an automatic way. We have a list to start with some automatic aspects (instruments filtered with pictures); can delete or filter out some of the problems. 
    _Presentation on generating this list (and problems inherent therein) for the LinkedMusic project meeting?_
Plan to take the id’s from spreadsheet with the pictures and numbers (if exists) put up on site; may discuss further. 
More interesting to talk about how things are organized (or not organized). Numbers: if you have X filters you get Y number of instruments. "Some sense of how [redacted] it is."
Main task of this project is cleaning up the music part of wikiData, be care-taker.

**Some discussion of how to put things up:**
-build a template with thumbnails (150x150?)
-thumbnails kept at wikimedia commons (not stored)
-may cause problems with standardization; will see
-may already have some API for this on wikipedia


**Data structure and backend implementation:**

   _What is the status of implementation of the agreed upon models?_
Had discussion of taxonomy (leave out), capturing sources of names
updates from @dchiller (see also (https://github.com/DDMAL/VIM/pull/37))
-Merged an initial dockerized Django instance in the GitHub repo (https://github.com/DDMAL/VIM/pull/25)
-to do: working on populating database with initial instruments so there is a backend to display in the front end. (https://github.com/DDMAL/VIM/issues/28)
-Instrument part is pretty much done

-Looked into languages on Wikidata, which are also complicated/messy (https://github.com/DDMAL/VIM/issues/27). 
Wikidata knows about 7500 languages (e.g. have property P424, language) but not all  (500? 700?)are supported by Wikidata 
Can't just label and give ETF code bc there are restrictions on the key/ID
-Started with just English/French for now.
-reminder of existence of Noto ("no tofu") font (https://github.com/DDMAL/VIM/discussions/20)
-discussion of using "also known as" tags; deemed redundant given image/QID
-How to display names. Under the picture? Mouseover? Can overly long names wrap with Bootstrap?


**October Meeting Discussion:**

User stories  https://github.com/orgs/DDMAL/projects/28/views/1 converted to issues:
-Need a database in Django importing Kyrie’s spreadsheet  https://github.com/DDMAL/VIM/issues/28
-Need the gallery https://github.com/DDMAL/VIM/issues/29 , https://github.com/DDMAL/VIM/issues/30
-Need to be able to select a language. (In this case French), choose to see labels in that language https://github.com/DDMAL/VIM/issues/31


**Next:**
-create details view? https://github.com/DDMAL/VIM/issues/32  
-But what are those details? https://github.com/DDMAL/VIM/issues/33 
-presumably click, see AV resources, other language names, add name--an embedded wikidata subset
-does this include what we keep or things from Wikidata--if so, what? can we just link to WikiData?
-Probably eventually want the user to be able to decide how many properties to include
-longterm, want people to be able to add a name while still in VIM.
-Decision: for now this should open a new page with an image and name(s).

-“I want to request a json object” : removed as the Django template will request it

-“I want to see contents of VIM in my language”: will use google translate widget (discussion of how to find out what users want among the ~30 choices on google translate)
-note the distinction of "language they see the site in" vs "language of the instrument names"

-Front end to work on /assign some of these issues

Several post-meeting issues created:
-adding geography data to models https://github.com/DDMAL/VIM/issues/34
-adding name/image info to models https://github.com/DDMAL/VIM/issues/35
-adding a user model https://github.com/DDMAL/VIM/issues/36

**Reply by dchiller** [reply]
**Agenda for 28 Sept**

1. Status of instrument gallery view.

> - Provide thumbnails (150 x 150)
> - Store images on Wikimedia Commons for now
> - Paginate results
> - What is the best way to display names for now?

2. Status of instrument detail view.

> - User is able to click on instrument tile in gallery view
> - Is shown any additional details. For now: link to Wikidata, image, and various names.

3. VIM-related presentations at the LinkedMusic October meeting. See discussion at https://github.com/DDMAL/linkedmusic-datalake/discussions/19.

>  - Finding instruments on Wikidata
>  - Show-and-tell basic VIM demo

4. Next steps.

> - Finish modeling? (Geography, languages, sources)
> - Add user support?
> - Integrate with Google translate?

5. Anything else?

**Comment by maregold** [comment]
VIM 9/28
1. Status of instrument gallery view.
get the instrument page hosted somewhere 
having placeholders is fine
Provide thumbnails (150 x 150) - possibly change to having all instruments have a max width and height 

3. VIM-related presentations at the LinkedMusic October meeting. See discussion at
 [Potential talks/discussions at Oct LinkedMusic Project Meeting linkedmusic-datalake#19](https://github.com/DDMAL/linkedmusic-datalake/discussions/19).

4. Next steps.
deployment is priority one
some not available / please fill this in if we don't have the name in a target language 
most people in the world will be able to read a language in google translate

**Reply by dchiller** [reply]
**Agenda for 5 October**

1. Deployment status update

2. Frontend status update

3. Anything else?

**Comment by maregold** [comment]
VIM 10/5/23
1. Deployment status update
- dylan getting code ready for deployment, just got access today
- wanyi also helping
- vim.staging.simssa
1. Frontend status update
- can now toggle image sizes for instruments 
- need to decide on the logo
- about - just put a couple of sentences
- get translate button to work
- pagination specificities 
- some images don't show up
1. Anything else?
- implementing hierarchy 
- free text search - indexing wikipedia if wikidata does not have
	 all pages should work in english and french for now
	 the text is only to power the search, maybe have a pointer to wikipedia somewhere, but the user should not see the actual description

**Reply by dchiller** [reply]
**Agenda: 12 October**

1. Front-end updates
  - translation
  - pagination
2. Search updates
  - Hornbostel-Sachs faceted search
  - Wikipedia-enabled full text search
3. Deployment updates
4. Anything else?

**Comment by annamorphism** [comment]
Meeting notes, Oct.12:
Front-end updates:
-Site was down and throwing 500 error (some issue in deployment on staging?) but working locally
-unrelated to "fix nginx port mapping" issue https://github.com/DDMAL/VIM/issues/62
-probably caused by many recent changes
-led to --> https://github.com/DDMAL/VIM/issues/71
-would like to have a production server so that there is a stable version (https://github.com/DDMAL/VIM/issues/63)


Updates on translation and texts:
-Detail view includes a placeholder text for now (a la MIMO); what should this be in future?
-decision not to have this/hide this; possibly point to wikipedia down the road for disambiguation needs
-detail view could take user to wikidata page or instrument-AVresource
-Use Google API for site text (NOT instruments names) using tags (like LinkedMusic homepage) #73 (also led to #74, #75)
-also noted to streamline log-in/sign-up buttons #65 


Pagination updates:
-Can now set number of instruments per page (using slider) and go to a specific page (c.f. #49)
-Discussion of how this is done on other sites (amazon, booking.com) (usually just "show more")--may modify this


Search updates:
-looking into both faceted search using Hornbostel-Sachs (#55) and Wikipedia-enabled full-text search (#54)
-started with HS; since each digit place is some class, might have e.g. “at most 4 facets" to choose the number of nests in the faceted search. (Second level has 9, all the way down is 306...)
-decision to stop at 9 since this is just one possible facet.
-Wikipedia search still very much in the future
-after meeting the search box/tab was also improved (#70)

Deployment updates:
-502 error--a problem with the code rather than server?

Other updates:
-update on tiff display: not supported by modern browsers except safari. (Maybe need on the fly conversion? #53 )
-update that instrument list has another ~500 items to be added
-issue created for displaying unnamed instruments #67

**Reply by dchiller** [reply]
**Agenda 19 October**

Server Updates
- To-Do: #71  
- #63 is complete

Front-end updates
- New language selectors (on staging server)
- Hornbostel-Sachs faceting (not yet merged)
- Pagination status?
- Others?

New issues
- #53 
- #67 

Anything else?

**Comment by PythonSemicolon** [comment]
LinkedMusic Meeting Notes - 22/10/2023

Use of facets to filter instrument view. Characteristics such as:
- plucked/bowed/struck
- material
- non-hierarchical facets in the form of tags

Alternatively: facets should be hierarchical -> layperson terms should not be on left facets, but maybe in a previous menu/page

Several mentions of image search

Naming: Thesaurus/concordance instead of museum?
- need to distinguish from MIMO

**Reply by maregold** [reply]
-	Remove the hierarchy aspect and flatten it to the basic sections.
o	Tag everything depending on different classifications.
o	Means of sound production, material, etc.
-	Stuff on the left must be big facets, complete, distinct categories.
-	If we want to derive properties, where do we get that information?
-	Image search – then you wouldn’t need a classification system.
o	Make that the main priority and focus on text/tag search later 
-	Think about how it's going to be different from MIMO 
-	Image search would make it more distinct

**Reply by dchiller** [reply]
More meeting notes
- Facet by tags — don't have a single classification system
- UI thought: When things are on the left — has to have big facet values and it really has to narrow it down - have to have a complete and distinct system

**Comment by maregold** [comment]
VIM 10/19

Front-end updates

- New language selectors (on staging server)
	- Google Translate has been implemented for the webpage
		- has an instructional badge as well
	- French and English for the instrument list only so far
	- LinkedMusic website should have the same interface as VIM for Google Translate
- Hornbostel-Sachs faceting (not yet merged)
	- First level is done
	- Leave as uncategorized / unclassified for those without classification on Wikidata
	- "Search in" (Wikidata) facets/selectors
- Pagination status?
	- Wikidata pagination example + page numbers
- Others?
	- Added LinkedMusic logo (make it even with DDMAL logo)

**Reply by dchiller** [reply]
**Agenda for 26 Oct**

1. Review notes from LinkedMusic project meeting
     1. See above at https://github.com/DDMAL/VIM/discussions/11#discussioncomment-7351050
2. Development updates
     1. Pagination changes (h/t @Monalisa98927)
     2. DevOps pipeline (h/t @softcat477)

**Comment by maregold** [comment]
1. See above at [Meeting notes #11 (comment)](https://github.com/DDMAL/VIM/discussions/11#discussioncomment-7351050)
	- The HBS numbering system has a layering system if we want to do plucked/bowed etc. 
		- Violin is categorized as bowed
	- Kyrie will create a table of HBS numbers that maps to categories that can be used as facets - then Kun will implement that table into the left-hand side of the instrument list
	- Don't want to keep a local copy of our tags
	- Need to show you somewhere what facet is active
	- The facets need to be translatable 
	- Image search will happen one day, but the images we have now aren't useful 
		- Google Lens API
	- Show me the instruments that don't have a name in language X
	- Wikidata Professional Development Training Module 
- Development updates
    1. Pagination changes 
	    1. Unhighlight the page number you are on
	    2. Work on login next - copy from Cantus
    2. DevOps pipeline

**Reply by dchiller** [reply]
**Agenda for 2 November**

1. Issues closed in last week
    - #87 
    - #84 
2. HBS <-> facet categories mapping update
> Kyrie will create a table of HBS numbers that maps to categories that can be used as facets - then Kun will implement that table into the left-hand side of the instrument list
3. Upcoming priorities

**Reply by dchiller** [reply]
Add and discuss names in #88! 

Nominations for new names for VIM end 31 December AOE.

**Reply by dchiller** [reply]
I find MDN resource for accessibility pretty good and helpful: https://developer.mozilla.org/en-US/docs/Web/Accessibility
